testers/python/server/markus_pyta_tester.py

```diff
@@ -28,8 +28,10 @@ class MarkusPyTATest(MarkusTest):
         'reported': "{} error(s)"
     }
 
-    def __init__(self, tester, test_file, data_files, points, test_extra, feedback_open=None):
-        super().__init__(tester, test_file, data_files, points, test_extra, feedback_open)
+    def __init__(self, test_file, data_files, points, test_extra, feedback_open=None, **kwargs):
+        super().__init__(test_file, data_files, points, test_extra, feedback_open)
+        self.pyta_config = kwargs['pyta_config']
+        self.devnull = kwargs['devnull']
```

Comment:
These will raise a KeyError if there are no `pyta_config` or `devnull` keyword argument. Either give these a default value or check if this key exists before assigning it.